tests/common.py

```diff
@@ -196,3 +196,85 @@ def __getitem__(self, index: int):
             return self.transform(x), y
         else:
             return x, y
+
+
+def dummy_tensor_batch(batch_size=12) -> torch.Tensor:
```

Comment:
Can these be moved  to the `test_split_batch` directly? They aren't "common" enough to be worth centralizing just yet.